skfda/ml/_neighbors.py

```diff
@@ -0,0 +1,1665 @@
+"""Module with classes to neighbors classification and regression."""
+
+from abc import ABCMeta, abstractmethod, abstractproperty
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin, RegressorMixin
+from sklearn.utils.validation import check_is_fitted as sklearn_check_is_fitted
+from sklearn.utils.multiclass import check_classification_targets
+from sklearn.preprocessing import LabelEncoder
+
+# Sklearn classes to be wrapped
+from sklearn.neighbors import NearestNeighbors as _NearestNeighbors
+from sklearn.neighbors import KNeighborsClassifier as _KNeighborsClassifier
+from sklearn.neighbors import (RadiusNeighborsClassifier as
+                               _RadiusNeighborsClassifier)
+from sklearn.neighbors import KNeighborsRegressor as _KNeighborsRegressor
+from sklearn.neighbors import (RadiusNeighborsRegressor as
+                               _RadiusNeighborsRegressor)
+
+from .. import FDataGrid
+from ..misc.metrics import lp_distance, pairwise_distance
+from ..exploratory.stats import mean
+
+__all__ = ['NearestNeighbors', 'KNeighborsClassifier',
```

Comment:
If the module is private, this is unnecessary (but you can keep it for documentation).

Reply:
Removed.